### PR TITLE
feat(HACBS-663): add milestone 5 demo resources

### PR DESCRIPTION
**Pull request by johnbieren**
Signed-off-by: Johnny Bieren <jbieren@redhat.com>

### CODE REVIEW

File: RUNNING_THE_DEMO.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Milestone 5 demo
```
Comment (johnbieren) [inline RUNNING_THE_DEMO.md]: @davidmogar `README.md` is something else. WDYT of this name instead for this?

Comment (davidmogar) [reply RUNNING_THE_DEMO.md]: I think we should integrate both. I like the current `README.md`, but maybe we could expand it with `## Running the demo` stuff. Maybe we could even rename the `## Tags` heading to something like `Searching for previous milestone demos?` so it's more clear.

Comment (johnbieren) [reply RUNNING_THE_DEMO.md]: Sure. Done

File: README.md [hunk 2]
```diff
@@ -4,10 +4,27 @@ This repository contains the yaml files used for the release workstream HACBS mi
...
-## Tags
+## Searching for previous demos
 
 Milestones are separated in this repository by tags.
 To view the demo resources for a previous milestone, simply checkout its tag.
 Be warned, however, that we make no guarantee that the past demo resources work with the latest HACBS deployment.
 Once a milestone passes, we will only be focusing on demo resources for the next milestone.
 So, if you need working demo resources, use the `main` branch.
+
+## Running the current demo (milestone 5)
```
Comment (davidmogar) [inline README.md]: I'd probably remove any mention to the milestone. Ideally, with time, the `README.md` shouldn't need a change per milestone.

**Review by davidmogar — APPROVED** [review]

**Pull request merged**